spec/factories/rents.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :rent do
+    user { user_id }
```

Comment:
Have your factory be able to create all it needs when building the model, so if you just need a rent and you don't care about the user, you only call the rent factory, and it takes care of all the relations